magento/magento2
# Resolve Mass Delete Widget should have "Confirmation Modal"

### Description (*)


 Mass Delete Widget should have "Confirmation Modal" like that:

![image](https://user-images.githubusercontent.com/8234209/69551223-ea7d9600-0fce-11ea-8b5a-879d8ef978d0.png)

We can see it is already in Functional Test (dev/tests/functional/tests/app/Magento/Widget/Test/TestCase/DeleteWidgetEntityTest.php), but it doesn't have the code to do the business.

![image](https://user-images.githubusercontent.com/8234209/69551387-2d3f6e00-0fcf-11ea-9cd5-b2982b39fce8.png)

=> We should open Confirmation Modal when delete widget by mass action.


### Fixed Issues (if relevant)



### Manual testing scenarios (*)

1. Go to backend, create some widget
2. Select and choose mass action Delete widget.
3. Check the modal:

![image](https://user-images.githubusercontent.com/8234209/69551223-ea7d9600-0fce-11ea-8b5a-879d8ef978d0.png)

### Questions or comments


### Contribution checklist (*)
